src/pages/home/owners/OwnersTable.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableCellProps,
+  TableContainer,
+  TableContainerProps,
+  TableFooter,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import theme from "theme";
+import { useEffect, useState } from "react";
+import { Typography } from "elements";
+import { useWindowDimensions } from "common";
+import { StyledPagination } from "components";
+import PendingBadge from "assets/images/PendingBadge";
+import { Owner } from "./mockOwnersData";
+interface OwnersTableProps {
+  ownersData: Owner[];
+}
+const StyledTableContainer = styled(TableContainer)<TableContainerProps>`
+  border-color: ${theme.colors.grey500};
+`;
+const StyledTableCell = styled(TableCell)<TableCellProps>`
+  border-color: ${theme.colors.grey500};
+  color: ${theme.colors.grey100};
+  height: 60px;
+`;
+
+export default function OwnersTable({ ownersData }: OwnersTableProps) {
+  const rowHeight = 60;
+  const [page, setPage] = useState(1);
+  const [rowsPerPage, setRowsPerPage] = useState(0);
+  // Used to avoid a layout jump when reaching the last page with empty rows.
+  const emptyRows =
+    page > 1 ? Math.max(0, page * rowsPerPage - ownersData.length) : 0;
+  const lastRowOnPage = (page - 1) * rowsPerPage + rowsPerPage;
+
+  // determines how many rows to display per page
+  const windowHeight = useWindowDimensions()?.height;
+  useEffect(() => {
+    setRowsPerPage(
+      windowHeight
+        ? Math.floor((windowHeight - 245 - rowHeight - 60 - 30) / rowHeight)
+        : 5
+    );
+  }, [windowHeight]);
+
+  const handlePageChange = (
+    event: React.ChangeEvent<unknown>,
+    page: number
+  ) => {
+    setPage(page);
+  };
+
+  return (
+    <StyledTableContainer>
+      <Table sx={ { minWidth: 500 } } aria-label="Owners pagination table">
+        <TableHead sx={ { lineHeight: "16.94px" } }>
```

Comment:
Curious here as to the specific line height value, the line height should be set  similar to our typography values